develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.Exceptions.Exception.md

```diff
@@ -4,7 +4,7 @@ uid: Protocol.Params.Param.Interprete.Exceptions.Exception
 
 # Exception element
 
-Defines an exception for an exception value you want to intercept of which the Interprete.Type is identical to that of the parameter in question..
+Defines an exception for an exception value you want to intercept of which the Interprete. Type is identical to that of the parameter in question.
```

Comment:
"Interprete.Type" is in fact correct, as this is the way the Type subtag of the Interprete tag is indicated. I'll revert that change but will keep the other correction.